serializer.rst

```diff
@@ -249,10 +249,14 @@ A service leveraging `APCu`_ (and APC for PHP < 5.5) is built-in.
         $container->loadFromExtension('framework', array(
             // ...
             'serializer' => array(
-                'cache' => 'serializer.mapping.cache.apc',
+                'cache' => 'serializer.mapping.cache.doctrine.apc',
             ),
         ));
 
+.. versionadded:: 3.1
+    The ``serializer.mapping.cache.doctrine.apc`` service was introduced in 3.1
```

Comment:
The `serializer.mapping.cache.doctrine.apc` service is now also deprecated since Symfony 3.2 (see symfony/symfony#19236).